transaction.go

```diff
@@ -27,7 +27,7 @@ func (tx *snowflakeTx) Rollback() (err error) {
 	if tx.sc == nil || tx.sc.rest == nil {
 		return driver.ErrBadConn
 	}
-	_, err = tx.sc.exec(context.TODO(), "ROLLBACK", false, false, nil)
+	_, err = tx.sc.exec(context.TODO(), "ROLLBACK", false, false, false, nil)
```

Comment:
If think you either add comments to the boolean parameters or change the method to clarify what parameters are. In Java, we usually create a builder to set each of the parameter individually with names and make a function call in the end.

Reply:
My point is there is two many `false` in the parameters. It is not clear which one is for what by seeing this line.